package/Shaders/ISReflectionsRayTracing.hlsl

```diff
@@ -142,18 +156,18 @@ PS_OUTPUT main(PS_INPUT input)
 		;  // Adjust based on performance/quality tradeoff
 
 	for (; iterationIndex < maxIterations; iterationIndex++) {
-		float3 iterationUvDepthDR = uvDepthStartDR + (iterationIndex / (float)maxIterations) * deltaUvDepthDR;
-		float3 iterationUvDepthSampleDR =
+		float3 iterationUvDepthDR = ViewToUVDepthHelper(lerp(csStart.xyz, csFinish.xyz, (iterationIndex / (float)maxIterations) * SSRParams.x * rcp(length(deltaUvDepth.xy))), eyeIndex);
+		float3 iterationUvDepthSampleDR = iterationUvDepthDR;
```

Comment:
Not sure why you had to break VR for this. The assignment is still there below at 170.

Reply:
Sorry this should have been written as a draft.